packages/react/src/alert/Alert.css.ts

```diff
@@ -71,14 +47,23 @@ export const icon = recipe({
       mt: "4",
       w: "auto",
     },
+    style({
+      color: iconColorVar,
+    }),
```

Comment:
We can just use sprinkles in the variant, I think there's no need of keeping it in `var`